cpp/src/parquet/encoding.cc

```diff
@@ -466,6 +466,15 @@ struct DictEncoderTraits<FLBAType> {
 // Initially 1024 elements
 static constexpr int32_t kInitialHashTableSize = 1 << 10;
 
+int RlePreserveBufferSize(int num_values, int bit_width) {
+  // Note: because of the way RleEncoder::CheckBufferFull() is called, we have to
+  // reserve
```

Comment:
The format of comment looks strange.

Reply:
Pasted from `DictEncoder`, let me change it